guillotina/api/content.py

```diff
@@ -583,6 +584,13 @@ def getRequestMethod(self):  # noqa
         raise ErrorResponse(
             "RequiredParam", _("Invalid params"), reason=error_reasons.REQUIRED_PARAM_MISSING, status=412
         )
+    except HTTPConflict:
+        raise ErrorResponse(
+            "Conflict Error",
+            "Id {data.get('new_id')} already exists in destination data.get('destination')",
+            reason=error_reasons.CONFLICT_ID,
+            status=409,
+        )
```

Comment:
You don't need to capture the `HTTPConflict` and raise a `ErrorResponse`, the HTTPConflict isn't used to retry request (only `ConflictError` and `TIDConflictError` are retried): https://github.com/plone/guillotina/blob/fd73ae1bbe738d57eaa812b7eefe81d2914048ea/guillotina/asgi.py#L101-L108

Reply:
I see, thanks for the explanation :), fixing...